docs/UserGuide.md

```diff
@@ -106,22 +106,22 @@ Shows a list of all employees in TaskHub.
 
 Format: `listE`
 
-### Editing a person : `edit`
+### Editing a employee : `edit`
 
-Edits an existing person in the address book.
+Edits an existing employee in the TaskHub.
 
 Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
 
```

Comment:
There seem to be some occurrences such as the one above of the older command format (e.g. `editE`) but we can fix those in a separate PR.